--- OutlinesTT.sfd	2014-06-23 22:46:23.612115000 +0900
+++ OutlinesG.sfd	2014-06-23 23:00:47.578598000 +0900
@@ -307,7 +307,7 @@
 EndPrivate
 TeXData: 1 0 0 346030 173015 115343 0 1048576 115343 783286 444596 497025 792723 393216 433062 380633 303038 157286 324010 404750 52429 2506097 1059062 262144
 AnchorClass2: "viramamark"  "Below-base Mark to Mark-1" "ukaaramark"  "Below-base Mark to Mark-1" "bindumark"  "Above-base Mark to Mark-1" "ukaara"  "Below-base Marks-1" "virama"  "Below-base Marks-1" "ekaara"  "Above-base Marks-1" "iikaara"  "Above-base Marks-1" "aakaara"  "Above-base Marks-1" "repha"  "Above-base Marks-1" "bindu"  "Above-base Marks-1" 
-BeginChars: 1413 1413
+BeginChars: 1485 1485
 
 StartChar: akAra
 Encoding: 0 2309 0
@@ -100408,5 +100408,725 @@
 Fore
 Refer: 1137 95 N 1 0 0 1 0 852 2
 EndChar
+
+StartChar: u11580
+Encoding: 1413 71040 1413
+Width: 1687
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 0 2309 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u11581
+Encoding: 1414 71041 1414
+Width: 1841
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 1 2310 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u11582
+Encoding: 1415 71042 1415
+Width: 1726
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 2 2311 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u11583
+Encoding: 1416 71043 1416
+Width: 1595
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 3 2312 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u11584
+Encoding: 1417 71044 1417
+Width: 1566
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 4 2313 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u11585
+Encoding: 1418 71045 1418
+Width: 1898
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 5 2314 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u11586
+Encoding: 1419 71046 1419
+Width: 1357
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 6 2315 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u11587
+Encoding: 1420 71047 1420
+Width: 1626
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 7 2400 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u11588
+Encoding: 1421 71048 1421
+Width: 1421
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 8 2316 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u11589
+Encoding: 1422 71049 1422
+Width: 1482
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 9 2401 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u1158A
+Encoding: 1423 71050 1423
+Width: 1861
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 10 2319 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u1158B
+Encoding: 1424 71051 1424
+Width: 1861
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 11 2320 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u1158C
+Encoding: 1425 71052 1425
+Width: 1566
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 12 2323 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u1158D
+Encoding: 1426 71053 1426
+Width: 1898
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 13 2324 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u1158E
+Encoding: 1427 71054 1427
+Width: 1554
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 17 2325 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u1158F
+Encoding: 1428 71055 1428
+Width: 1830
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 18 2326 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u11590
+Encoding: 1429 71056 1429
+Width: 1830
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 19 2327 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u11591
+Encoding: 1430 71057 1430
+Width: 1732
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 20 2328 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u11592
+Encoding: 1431 71058 1431
+Width: 1658
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 21 2329 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u11593
+Encoding: 1432 71059 1432
+Width: 1376
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 22 2330 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u11594
+Encoding: 1433 71060 1433
+Width: 1665
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 23 2331 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u11595
+Encoding: 1434 71061 1434
+Width: 1658
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 24 2332 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u11596
+Encoding: 1435 71062 1435
+Width: 1357
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 25 2333 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u11597
+Encoding: 1436 71063 1436
+Width: 1708
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 26 2334 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u11598
+Encoding: 1437 71064 1437
+Width: 1644
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 27 2335 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u11599
+Encoding: 1438 71065 1438
+Width: 1683
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 28 2336 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u1159A
+Encoding: 1439 71066 1439
+Width: 1296
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 29 2337 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u1159B
+Encoding: 1440 71067 1440
+Width: 1871
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 30 2338 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u1159C
+Encoding: 1441 71068 1441
+Width: 1818
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 31 2339 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u1159D
+Encoding: 1442 71069 1442
+Width: 1290
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 32 2340 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u1159E
+Encoding: 1443 71070 1443
+Width: 1513
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 33 2341 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u1159F
+Encoding: 1444 71071 1444
+Width: 1480
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 34 2342 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115A0
+Encoding: 1445 71072 1445
+Width: 1671
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 35 2343 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115A1
+Encoding: 1446 71073 1446
+Width: 1409
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 36 2344 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115A2
+Encoding: 1447 71074 1447
+Width: 1665
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 37 2346 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115A3
+Encoding: 1448 71075 1448
+Width: 1677
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 38 2347 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115A4
+Encoding: 1449 71076 1449
+Width: 1290
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 39 2348 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115A5
+Encoding: 1450 71077 1450
+Width: 1634
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 40 2349 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115A6
+Encoding: 1451 71078 1451
+Width: 1609
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 41 2350 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115A7
+Encoding: 1452 71079 1452
+Width: 1687
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 42 2351 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115A8
+Encoding: 1453 71080 1453
+Width: 1179
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 43 2352 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115A9
+Encoding: 1454 71081 1454
+Width: 1646
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 44 2354 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115AA
+Encoding: 1455 71082 1455
+Width: 1216
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 45 2357 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115AB
+Encoding: 1456 71083 1456
+Width: 1830
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 46 2358 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115AC
+Encoding: 1457 71084 1457
+Width: 1665
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 47 2359 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115AD
+Encoding: 1458 71085 1458
+Width: 1843
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 48 2360 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115AE
+Encoding: 1459 71086 1459
+Width: 1683
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 49 2361 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115AF
+Encoding: 1460 71087 1460
+Width: 163
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 53 2366 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115B0
+Encoding: 1461 71088 1461
+Width: 335
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 57 2367 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115B1
+Encoding: 1462 71089 1462
+Width: 368
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 59 2368 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115B2
+Encoding: 1463 71090 1463
+Width: 0
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 64 2369 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115B3
+Encoding: 1464 71091 1464
+Width: 0
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 69 2370 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115B4
+Encoding: 1465 71092 1465
+Width: 0
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 74 2371 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115B5
+Encoding: 1466 71093 1466
+Width: 0
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 76 2372 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115B8
+Encoding: 1467 71096 1467
+Width: 0
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 77 2375 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115B9
+Encoding: 1468 71097 1468
+Width: 0
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 79 2376 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115BA
+Encoding: 1469 71098 1469
+Width: 0
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 81 2379 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115BB
+Encoding: 1470 71099 1470
+Width: 0
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 82 2380 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115BC
+Encoding: 1471 71100 1471
+Width: 0
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 15 2305 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115BD
+Encoding: 1472 71101 1472
+Width: 0
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 14 2306 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115BE
+Encoding: 1473 71102 1473
+Width: 430
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 16 2307 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115BF
+Encoding: 1474 71103 1474
+Width: 0
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 83 2381 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115C0
+Encoding: 1475 71104 1475
+Width: 0
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 880 2364 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115C1
+Encoding: 1476 71105 1476
+Width: 1839
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 96 182 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115C2
+Encoding: 1477 71106 1477
+Width: 1316
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 97 2404 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115C3
+Encoding: 1478 71107 1478
+Width: 1544
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 98 2405 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115C4
+Encoding: 1479 71108 1479
+Width: 675
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 863 58819 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115C5
+Encoding: 1480 71109 1480
+Width: 614
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 864 58820 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115C6
+Encoding: 1481 71110 1481
+Width: 1646
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 865 58821 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115C7
+Encoding: 1482 71111 1482
+Width: 1712
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 866 58822 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115C8
+Encoding: 1483 71112 1483
+Width: 1486
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 867 58823 N 1 0 0 1 0 0 2
+EndChar
+
+StartChar: u115C9
+Encoding: 1484 71113 1484
+Width: 1720
+VWidth: 0
+Flags: W
+LayerCount: 2
+Fore
+Refer: 877 58833 N 1 0 0 1 0 0 2
+EndChar
 EndChars
 EndSplineFont
